jobs/pull-kubernetes-federation-e2e-gce-gci.sh

```diff
@@ -47,7 +47,7 @@ export KUBE_FASTBUILD=true
 ./hack/jenkins/build.sh
 
 # Push federation images to GCS.
-./build-tools/push-federation-images.sh
+./build/push-federation-images.sh
```

Comment:
Why do you think so? I dont see push-federation-images.sh being called from anywhere else (or from hack/jenkins/build.sh).